deepaas/api/v1.py

```diff
@@ -164,21 +166,22 @@ def get(self):
 
     # Fill the test parser with the supported arguments. Different models may
     # have different arguments.
+    test_parser = copy.deepcopy(data_parser)
```

Comment:
Why is this copy being made rather than creating a new parser?

Reply:
The copy is made because all modules use the base parser (eg. `data_parser`) that contains a `files` field and a `urls` field (ie. those are common test args across all modules).